model/qti/ImportService.php

```diff
@@ -328,7 +328,7 @@ public function importQTIPACKFile($file, core_kernel_classes_Class $itemClass, $
                         }
                         
                         // replace uri if changed
-                        if ($auxPath != $info['uri']) {
+                        if ($auxPath != ltrim($info['uri'], '/')) {
```

Comment:
looks a good idea but I am just curious : by any chance; would you know when and where the / has been added ?